src/CompanyService.Models.Broker/Models/UserData.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace LT.DigitalOffice.UserService.Models.Broker.Models
+{
+    public class UserData
+    {
+        public Guid Id { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string MiddleName { get; set; }
+        public bool IsActive { get; set; }
+
```

Comment:
Why you use set? you have Create method to set data

Reply:
```
new UserData
{
    Id = _directorGuid,
    IsActive = true,
    FirstName = "Spartak",
    LastName = "Ryabtsev",
    MiddleName = "Alexandrovich"
};
```

Reply:
ok